xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -116,6 +116,7 @@ func (sc *ServerConfig) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// Authority represents an Authority for an xDS control plane server.
```

Comment:
Can we have a comment which is more helpful than this :)

Reply:
I cannot think of useful comment for this struct. It's usage is explained in the Config struct where it's used.

I added a sentence.